tests/integration_tests/changefeed_query_toml_api/run.sh

```
@@ -0,0 +1,300 @@
+# This case verifies the API-layer TOML support added for issue #4936:
+#   GET /api/v2/changefeeds/:id with `Accept: application/toml` returns the
+#   changefeed as TOML, while the default (or application/json) keeps JSON.
+# It mirrors the behaviors covered by the PR1 pytest suite (kebab-case keys,
+# config sections, human-readable durations/timestamps, JSON<->TOML field
+# correspondence, password redaction) and checks that the exported TOML config
+# is import-compatible by feeding it back through `changefeed create --config`.
+
+set -euo pipefail
+
+CUR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $CUR/../_utils/test_prepare
```

Shellcheck (0.11.0)

[info] Not following: ./../_utils/test_prepare was not specified as input (see shellcheck -x).

(SC1091)